### DISCUSSION TITLE
How to create an Identity user custom property that allows selecting multiple values from a list?

### DISCUSSION THREAD

**Discussion by PabloClaus** [abpframework/abp#18305] [Q&A]
Hi everyone. I have this code in `ModuleExtensionConfigurator` class.

```
 ObjectExtensionManager.Instance.Modules()
            .ConfigureIdentity(identity =>
            {
                identity.ConfigureUser(user =>
                {
                  user.AddOrUpdateProperty<int>(
                           "AreaId", 
                            property =>
                              {
                                property.UI.Lookup.Url = "/api/app/instalacion/area";
                                property.UI.Lookup.DisplayPropertyName = "nombre";
                                property.UI.Lookup.ResultListPropertyName = string.Empty;
                              }
                    );
.........
```
This is rendered correctly in the UI as a list of items. So far, so good. 

**Is there a way to configure it so that, from the list, it allows selecting multiple items and saving them in the user's extra properties?**

A user could have 1 to 15 items associated.

Any suggestions?

**Comment by realLiangshiwei** [comment]
Hi,

> Is there a way to configure it so that, from the list, it allows selecting multiple items and saving them in the user's extra properties?

No, the Module Entity Extensions system does not support this. you need to implement it yourself.
